common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -2869,6 +2869,14 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     @ZAttr(id=-1)
     public static final String A_registeredAddress = "registeredAddress";
 
+    /**
+     * Can be used in Mail Recall to make it more secure from spoof.
+     *
+     * @since ZCS 10.0.0
+     */
+    @ZAttr(id=4097)
+    public static final String A_secretKeyForMailRecall = "secretKeyForMailRecall";
```

Comment:
Please read the below lines at the start of this file. 
The name of the attribute should start with **zimbra**

```
Zimbra devs - things to keep in mind when adding new attributes:

 1) Always add the new attribute at the end of the list, adding one to
    the highest attr id (the one that corresponds to the OID) number
    in use.  Do not sort by name - it will break OID allocation.

 2) When defining new attributes, try and use descriptive names. The
    name should *always* start with zimbra*.  When defining
    multiple-attributes use a standard prefix for all related
    attributes. For example, if you are defining multiple attrs all
    related to the blob store, then prefix them all with zimbraBlob*.
```

Reply:
Corrected.